app/schemas/user_signup.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../models');
+
+const signUpSchema = {
+  firstName: {
+    exists: true,
+    notEmpty: true,
+    errorMessage: 'First name cannot be empty!'
+  },
+  lastName: {
+    exists: true,
+    notEmpty: true,
+    errorMessage: 'Last name cannot be empty!'
+  },
+  email: {
+    exists: true,
+    isEmail: { errorMessage: 'Not a valid email.' },
+    contains: { options: '@wolox.co', errorMessage: 'Email does not belong wolox domain.' },
```

Comment:
There are several wolox domains: wolox.(co|ar|cl|mx), find a way to accept all of these.